whipper/common/mbngs.py

```diff
@@ -161,10 +161,78 @@ def _getWorks(recording):
     if 'work-relation-list' in recording:
         for work in recording['work-relation-list']:
             if work['type-id'] in valid_work_rel_types:
-                works.append(work['work']['id'])
+                works.append(work['work'])
     return works
 
 
+def _getWorksId(works):
+    """Get work ids from "performance of" works"""
+    ids = []
+    for work in works:
+        ids.append(work['id'])
+    return ids
```

Comment:
(Writing this after my comment to `_getComposers()`, but basically same question. If the same Work is linked multiple times to a Recording, does this return a list of the same MBID multiple times, or… ?)

Reply:
I think it originally is gonna return the same thing but i will work on how to remove duplicate

Reply:
Or just use a set.

Reply:
I forced it with `list(set(ids))` i thought of using set in the beginning but maybe this way of adding a comment is easier to understand ?

Reply:
I would just use set in the beginning. Maybe with a comment, like `ids = set()  # using a set to prevent duplicates`. `list(set(ids))` seems wasteful, typecasting twice.

Reply:
okay agreed